includes/processors/class.llms.processor.achievement.sync.php

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
filename should use dashes instead of dots

Reply:
The processor class files instantiate the class and are not currently autoloaded. I'm saving that for later.

Reply:
I decided to modify `LLMS_Processors::load_processor()` to handle class file names with dashes.

@thomasplevy, does this change need a `.changelogs` files?

Reply:
NVM. It's a brand new file.